backend/LexBoxApi/GraphQL/OrgMutations.cs

```diff
@@ -89,6 +89,41 @@ public async Task<IQueryable<Organization>> AddProjectToOrg(
         return dbContext.Orgs.Where(o => o.Id == orgId);
     }
 
+    [Error<DbError>]
+    [Error<NotFoundException>]
+    [UseMutationConvention]
+    [UseFirstOrDefault]
+    [UseProjection]
+    public async Task<IQueryable<Organization>> AddProjectsToOrg(
+        LexBoxDbContext dbContext,
+        IPermissionService permissionService,
+        [Service] ProjectService projectService,
+        Guid orgId,
+        IEnumerable<Guid> projectIds)
+    {
+        var org = await dbContext.Orgs.Include(o => o.Members).Include(o => o.Projects).SingleOrDefaultAsync(o => o.Id == orgId);
+        NotFoundException.ThrowIfNull(org);
+        permissionService.AssertCanAddProjectToOrg(org);
+        // First make sure ALL projects pass permissions tests
+        foreach (var projectId in projectIds)
+        {
+            await permissionService.AssertCanManageProject(projectId);
+        }
+        // Now exclude any projects that don't actually exist or the org already has or that don't exist
+        var alreadyInOrg = org.Projects.Select(o => o.Id).ToHashSet();
+        var filteredIds = projectIds.Where(id => !alreadyInOrg.Contains(id));
+        var updates = filteredIds.Select(projectId => new OrgProjects { OrgId = orgId, ProjectId = projectId });
+        dbContext.OrgProjects.AddRange(updates);
+        dbContext.Projects.Where(p => filteredIds.Contains(p.Id)).ExecuteUpdate(p => p.SetProperty(p => p.UpdatedDate, DateTime.UtcNow));
```

Comment:
should use ExecuteUpdateAsync instead

Reply:
I was wondering why there wasn't an async version of this, turns out I just completely missed it in the Intellisense list. Done.